src/ng/directive/ngModel.js

```diff
@@ -599,7 +596,7 @@ var NgModelController = ['$scope', '$exceptionHandler', '$attrs', '$element', '$
       forEach(ctrl.$asyncValidators, function(validator, name) {
         var promise = validator(modelValue, viewValue);
         if (!isPromiseLike(promise)) {
-          throw $ngModelMinErr("$asyncValidators",
+          throw minErr('ngModel')("$asyncValidators",
```

Comment:
This is actually a separate problem, too: there's no error page for "$asyncValidators" (and the err code doesn't conform to our standards)